spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/netty/NettyReactiveWebServerFactoryTests.java

```diff
@@ -101,14 +99,6 @@ public void whenSslIsConfiguredWithAValidAliasARequestSucceeds() {
 		StepVerifier.create(result).expectNext("Hello World").verifyComplete();
 	}
 
-	@Test
```

Comment:
This test is no longer valid as the server will not even startup if its configured w/ an invalid alias.